xml/deploy_cephadm.xml

```diff
@@ -158,6 +158,18 @@
      Install the <literal>salt-master</literal> on the &smaster; node:
     </para>
 <screen>&prompt.smaster;zypper in salt-master</screen>
+   </step>
+   <step>
+     <para>
+       Edit <filename>/etc/salt/master</filename> and uncomment the following
+       line:
+     </para>
+<screen>#log_level: WARN</screen>
```

Comment:
I'm thinking we should change `log_level` to `log_level_logfile` here - @ricardoasmarques Do you agree?